static/src/javascripts/projects/common/modules/experiments/tests/next-in-series.js

```diff
@@ -64,6 +66,10 @@ define([
                         var el = $.create(render(series));
                         el.insertAfter($articleBody);
                     }
+                },
+
+                success: function(complete) {
+                    bean.on($('.next-in-series-test__remind-me-link')[0], 'click', complete);
```

Comment:
I think it's very unlikely but not guaranteed not to happen. I'll change it to work by checking if you're on the survey URL from the appropriate referrer instead